### ISSUE TITLE
Extension issue

### ISSUE THREAD

**Issue by bm-stschneider** [kisstkondoros/gutter-preview#37]
- Issue Type: `Bug`
- Extension Name: `vscode-gutter-preview`
- Extension Version: `0.17.2`
- OS Version: `Darwin x64 17.7.0`
- VSCode version: `1.32.0-insider`

```json 
{
	"activationTimes": {
		"startup": true,
		"codeLoadingTime": 112,
		"activateCallTime": 7,
		"activateResolvedTime": 19,
		"activationEvent": "*"
	},
	"runtimeErrors": [
		{
			"name": "TypeError [ERR_INVALID_ARG_TYPE]",
			"message": "The \"path\" argument must be of type string. Received type undefined"
		},
		{
			"name": "TypeError [ERR_INVALID_ARG_TYPE]",
			"message": "The \"path\" argument must be of type string. Received type undefined"
		},
		{
			"name": "TypeError [ERR_INVALID_ARG_TYPE]",
			"message": "The \"path\" argument must be of type string. Received type undefined"
		},
		{
			"name": "TypeError [ERR_INVALID_ARG_TYPE]",
			"message": "The \"path\" argument must be of type string. Received type undefined"
		}
	]
}
```

**Comment by kisstkondoros** [comment]
Thanks for reporting the issue.

It is related to https://github.com/kisstkondoros/gutter-preview/blob/master/src/extension.ts#L111
if there is no workspace folder the problem occurs, a simple null check should do it.

I'll release a fix for it at the evening, sorry for the inconvenience

**Comment by kisstkondoros** [comment]
Closed accidentaly

**Comment by kisstkondoros** [comment]
:D

**Comment by bm-stschneider** [comment]
It is a multi folder workspace.

**Comment by kisstkondoros** [comment]
Issue was hopefully fixed with the new version: 0.17.3. It is already available in the market.
